config/initializers/session_store.rb

```diff
@@ -1 +1,15 @@
-Rails.application.config.session_store :active_record_store, key: '_upaya_session'
+require 'idle_session_expiration_store'
+
+options = {
+  key: '_upaya_session',
+  redis: {
+    driver: :hiredis,
+    expire_after: Figaro.env.session_timeout_in_minutes.to_i,
```

Comment:
Looks like you accidentally removed the `.minutes` from the end here? I have this branch locally and it has the `.minutes` at the end. Otherwise, won't it expire every 8 seconds?